### ISSUE TITLE
Add reboot required notification for cmdlets that require it

### ISSUE THREAD

**Issue by mikenelson-io** [jaapbrasser/CustomizeWindows11#13]
Some, if not all (I have not tested all of them) of the changes made by the cmdlets do require a reboot or a restart of Explorer to take effect.

**Comment by mikenelson-io** [comment]
Ties into Issue #8, however, not all instances of restarting Explorer.exe will work, whereas a reboot surely will.

**Comment by jaapbrasser** [comment]
Indeed, good idea, have you noticed any cmdlets in particular that require it.

Although  I guess I'll have to go over all over them after I get the restart explorer built-in.
